tests/testthat/test-fetch-mailinglist.R

```diff
@@ -1,9 +1,10 @@
-test_that("fetch_mailinglist returns a tbl_df with expected column names and types", {
+skip_on_cran()
 
-  vcr::use_cassette("fetch_mailinglist", {
-    x <- fetch_mailinglist("ML_6xLATuSWeOcGWyh")
-  })
+test_that("fetch_mailinglist returns a tbl_df with expected column names and types", {
+  local_mocked_bindings(glue_api_v3 = function(base_url) "https://stoplight.io/mocks/qualtricsv2/publicapidocs/60928")
+  local_mocked_bindings(paginate_api_request = function(fetch_url) qualtrics_api_request("GET", url = fetch_url)$result$elements)
```

Comment:
The mock server for this endpoint does have something in `nextPage` so we also need to mock the paginating.